packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -54,6 +58,24 @@ export const getExplorerUrl = (chainId: ChainId) => {
   return explorerUrls[chainId] ?? explorerUrls[ChainId.Mainnet] //defaults to etherscan
 }
 
+export const getBlockTime = (chainId: ChainId) => {
+  const network = l1Networks[chainId]
+  if (!network) {
+    throw new Error(`Couldn't get block time. Unexpected chain ID: ${chainId}`)
```

Comment:
I'm curious, if we throw error here, are we able to recover gracefully, and still display the WithdrawalCountdown (without countdown)?

Reply:
Should never happen as we have a couple of checks before that, so I'm fine with just throwing here - would help raise some hidden bugs.